relay-server/src/actors/project_buffer.rs

```diff
@@ -380,6 +399,13 @@ impl BufferService {
         sender: &mpsc::UnboundedSender<ManagedEnvelope>,
     ) -> Result<(), QueueKey> {
         loop {
+            // By default this creates a prepared statement which is cached and re-used.
+            //
+            // Removing envelopes from the on-disk buffer in batches has following implicatations:
```

Comment:
```suggestion
            // Removing envelopes from the on-disk buffer in batches has following implications:
```

Reply:
(also adding "the":)
```suggestion
            // Removing envelopes from the on-disk buffer in batches has the following implications:
```